example/lib/main.dart

```diff
@@ -21,19 +21,20 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return StreamBuilder<InputDecorationTheme?>(
-        stream: inputDecorationThemeStream.stream,
-        initialData: null,
-        builder: (context, snapshot) {
-          return MaterialApp(
-            title: 'FHIR Questionnaire Demo',
-            scrollBehavior: const CustomScrollBehavior(),
-            theme: ThemeData(
-                colorScheme: ColorScheme.fromSeed(seedColor: Colors.green),
-                useMaterial3: true,
-                inputDecorationTheme: snapshot.data),
-            home: const MyHomePage(),
-          );
-        });
+      stream: inputDecorationThemeStream.stream,
+      initialData: null,
+      builder: (context, snapshot) {
+        return MaterialApp(
+          title: 'FHIR Questionnaire Demo',
+          scrollBehavior: const CustomScrollBehavior(),
+          theme: ThemeData(
+              colorScheme: ColorScheme.fromSeed(seedColor: Colors.green),
+              useMaterial3: true,
+              inputDecorationTheme: snapshot.data),
+          home: const MyHomePage(),
+        );
+      },
+    );
```

Comment:
This seems like a change in the format, run a `dart format .` to apply the default format.

Reply:
I added a `,` betweek `})`. which changed the style after format. do you want to revert it?